src/widgets/Typography/TextWithIcon.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable arrow-body-style */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text } from 'react-native';
+
+import { FlexRow } from '../FlexRow';
+
+import { APP_FONT_FAMILY, DARKER_GREY } from '../../globalStyles';
+
+const SIZES = {
```

Comment:
Come up a couple times now; would it be better to have a base `Text` component that is our own wrapper to the RN component, then use that here? That way it's more accessible to use the same constants for text throughout the app.

Reply:
I knew this would come up - I am happy if we create a new issue for typography consistency and I know this makes it slightly worse but we are are already behind schedule on the project and I don't believe we have time to be doing it.